<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Debasish" failures="0" tests="1" name="Default test" time="20.741" errors="0" timestamp="2024-05-15T21:24:11 IST">
  <testcase classname="in.debasish.NativeApp.BaseClass" name="@BeforeClass configuration" time="20.671">
    <failure type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException" message="The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\Debasish\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 0.0.0.0, --base-path, wd/hub/] 
Process output: [35m[Appium][39m Welcome to Appium v2.5.4
[35m[Appium][39m Non-default server args:
[35m[Appium][39m { basePath: [32m&amp;apos;wd/hub/&amp;apos;[39m }
[35m[Appium][39m The autodetected Appium home path: C:\Users\Debasish\.appium
[35m[Appium][39m Attempting to load driver uiautomator2...
[35m[Appium][39m Requiring driver at C:\Users\Debasish\.appium\node_modules\appium-uiautomator2-driver\build\index.js

">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\Debasish\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 0.0.0.0, --base-path, wd/hub/] 
Process output: [35m[Appium][39m Welcome to Appium v2.5.4
[35m[Appium][39m Non-default server args:
[35m[Appium][39m { basePath: [32m'wd/hub/'[39m }
[35m[Appium][39m The autodetected Appium home path: C:\Users\Debasish\.appium
[35m[Appium][39m Attempting to load driver uiautomator2...
[35m[Appium][39m Requiring driver at C:\Users\Debasish\.appium\node_modules\appium-uiautomator2-driver\build\index.js


at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:191)
at in.debasish.NativeApp.BaseClass.configuration(BaseClass.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4723/wd/hub/status] to be available after 20017 ms
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:89)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:156)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:178)
... 27 more
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:87)
... 29 more
]]>
    </failure>
  </testcase> <!-- @BeforeClass configuration -->
  <testcase classname="in.debasish.NativeApp.BaseClass" name="@AfterClass tearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase classname="in.debasish.NativeApp.LongPressEvent" name="longPressonPPlLink" time="0.0">
    <skipped/>
  </testcase> <!-- longPressonPPlLink -->
</testsuite> <!-- Default test -->
